src/Containers/Impl/Docker/LocalDockerContainer.cs

```diff
@@ -14,9 +17,32 @@ public class LocalDockerContainer : IContainer {
 
         public bool IsShortId => Id.Length < 64;
 
-        public LocalDockerContainer(string id, string name) {
-            Id = id;
-            Name = name;
+        public IEnumerable<int> HostPorts { get; }
+
+        internal LocalDockerContainer(JToken containerObject) {
+            Id = (string)containerObject["Id"];
+            Name = GetContainerName(containerObject);
+            HostPorts = GetHostPorts(containerObject);
+        }
+
+        private IEnumerable<int> GetHostPorts(dynamic containerObject) {
+            var hostPorts = new List<int>();
+            try {
+                dynamic portMap = containerObject.NetworkSettings.Ports["5444/tcp"];
```

Comment:
Does it mean that it is impossible to have more than one running container at the same time?